domains/measurements/Measurements.API/Tests/Extensions/NSubstituteExtensions.cs

```diff
@@ -0,0 +1,18 @@
+using Grpc.Core;
+using Grpc.Core.Testing;
+using NSubstitute;
+using NSubstitute.Core;
+using System.Threading.Tasks;
+
+namespace Tests.Extensions;
+
+public static class NSubstituteExtensions
+{
+    public static ConfiguredCall Returns<TResponse>(this AsyncUnaryCall<TResponse> value, TResponse response)
```

Comment:
Why is this needed?

Reply:
I can't get Returns to work with asynchronous calls wihtout it